src/KafkaFlow.Retry.API/AppBuilderExtensions.cs

```diff
@@ -18,34 +18,83 @@ public static class AppBuilderExtensions
                     .ApplicationServices
                     .GetService(typeof(IRetryDurableQueueRepositoryProvider)) as IRetryDurableQueueRepositoryProvider;
 
-            appBuilder.UseRetryEndpoints(retryDurableQueueRepositoryProvider);
+            appBuilder.UseRetryEndpoints(retryDurableQueueRepositoryProvider, string.Empty);
```

Comment:
Please replace this method implementation just calling the new method `UseKafkaFlowRetryEndpoints` with the parameter `endpointPrefix` to avoid duplicated code.